blank-canvas/languages/blank-canvas.pot

```diff
@@ -2,14 +2,14 @@
 # This file is distributed under the GNU General Public License v2 or later.
 msgid ""
 msgstr ""
-"Project-Id-Version: Blank Canvas 1.2.4\n"
+"Project-Id-Version: Blank Canvas 1.2.5-wpcom\n"
```

Comment:
Do we want to add `-wpcom` ? - I mean, I don't see an issue but before the update it wasn't there, so...

Reply:
I think it's supposed to be there now, so that it matches the current version on wp.com. (For early versions, we weren't using it, but we added it last week).  

I'll make sure that's removed for the WP.org submission.